cabby/data/metagraph/utils.py

```diff
@@ -15,20 +15,31 @@
 '''Utils for metagraph construction'''
 
 import networkx as nx
+import numpy as np
 import pandas as pd
 
-from typing import Dict, Tuple, Sequence, Text, Any
+import typing
 
-from cabby.geo.map_processing.map_structure import Map
-import cabby.geo.util as util
-from cabby.data.wikidata import query
-from cabby.geo.regions import Region
+from cabby.geo.map_processing import map_structure
+from cabby.data import wikidata
+from cabby.geo import regions
+from cabby.geo import util
+
+DEFAULT_POI_READABLE_NAME = "poi"
+
+# Type declarations
+Any = typing.Any
+Dict = typing.Dict
+Map = map_structure.Map
+Region = regions.Region
+Sequence = typing.Sequence
+Tuple = typing.Tuple
```

Comment:
Sorry, I should have mentioned that there is an exception for importing types from typing, so you can change this back to:

```
from typing import Any, Dict, Tuple
```

And then do the declarations for the other types.

https://google.github.io/styleguide/pyguide.html#typing-imports

Reply:
Fixed by 7eaff4a901306bf525964c94bc11200516130492